pkg/pillar/nireconciler/linux_config.go

```diff
@@ -636,6 +636,40 @@ func (r *LinuxNIReconciler) getIntendedNIL3Cfg(niID uuid.UUID) dg.Graph {
 			}, nil)
 		}
 	}
+	// Add statically defined routes into the NI routing table.
+	// No need to do this if GW is one of the apps and traffic from one app
```

Comment:
If 10.0.0.0/16 is routed out one NI and 10.0.2.0/24 is routed to the default NI's bridgeIP or a host on the default subnet, then you can't supress the /24 route, can you?

Reply:
Here we only suppress installation of routes inside the NI routing table _on the host_ when traffic is only forwarded by the bridge, not routed (all-ones netmask is disabled and route is between apps on the same subnet). In that case, the route would not be applied anyway. But we still propagate all the routes to the applications using DHCP.

Reply:
Ok